api/nodes/serializers.py

```diff
@@ -602,21 +604,33 @@ def create(self, validated_data):
         contributor.node_id = node._id
         return contributor
 
+
 class NodeContributorDetailSerializer(NodeContributorsSerializer):
     """
     Overrides node contributor serializer to add additional methods
     """
     id = ContributorIDField(required=True, source='_id')
+    index = ser.IntegerField(required=False, read_only=False)
+    # index = ser.IntegerField(required=False)
 
     def update(self, instance, validated_data):
+        index = None
+        if 'index' in validated_data:
+            index = validated_data.pop('index')
```

Comment:
Why `.pop`? `index = validated_data.get('index')` should do the same as these three lines.